argo_rollouts/hatch.toml

```diff
@@ -0,0 +1,13 @@
+[env.collectors.datadog-checks]
+
+[envs.default.env-vars]
+DDEV_SKIP_GENERIC_TAGS_CHECK = "true"
```

Comment:
Oh, so I just noticed this with argo workflows. `go.info` submits a `version` tag. Is that ok or should we rename it?